frontend/components/panels/WalletVatMkrDepositCheckPanel.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+        <WalletMkrDepositCheckPanel
+            :is-correct.sync="isEnoughMkrInWallet"
+            :wallet-mkr="walletMkr"
+            :wallet-vat-mkr="walletVatMkr"
+            :desired-amount="desiredAmount"
+            :network="network"
+            :token-address-mkr="tokenAddressMkr"
+            :disabled="disabled"
+            :is-loading="isLoading"
+            :is-explanations-shown="isExplanationsShown"
+            @refresh="$emit('refresh')"
+        />
+        <AllowanceAmountCheckPanel
+            :is-correct.sync="isAmountWithinAllowance"
+            :allowance-amount="allowanceAmount"
+            :desired-amount="desiredAmount"
```

Comment:
Also based on the screenshot: here we should pass the same value as displayed in `Minimum to deposit` field